src/NLog/Config/LoggingConfiguration.cs

```diff
@@ -182,6 +179,22 @@ public ReadOnlyCollection<Target> AllTargets
             }
         }
 
+        /// <summary>
+        /// Inserts NLog Config Variable without overriding NLog Config Variable assigned from API
+        /// </summary>
+        internal void InsertConfigFileVariable(string key, Layout value)
```

Comment:
While the config is most of the time a file, is file here really the right term?

Reply:
Will rename it to `InsertParsedConfigVariable`